homeassistant/util/timeout.py

```diff
@@ -16,7 +16,7 @@
 ZONE_GLOBAL = "global"
 
 
-class _State(str, enum.Enum):
+class _State(enum.Enum):
```

Comment:
This was a small cleanup I had already in the file, and since its only used internally, I left it in.